packages/app/src/specs/SpecNameDisplay.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div
+    class="max-w-60 truncate overflow-hidden"
+    data-cy="spec-filename"
+    :title="specFileName + specFileExtension"
+  >
+    <span class="font-semibold text-gray-800">{{ props.specFileName }}</span><span class="text-gray-600">{{ props.specFileExtension }}</span>
+  </div>
+</template>
+
+<script lang="ts" setup>
+
+const props = defineProps<{
```

Comment:
```suggestion
defineProps<{
```

If you don't have any computed values/need for props in the setup script, you can remove the `props` variable and reference the props directly (which is what you are doing on line 5). Line 7 could be refactored to just use `{{ specFileName }}`.